raiden/raiden_service.py

```diff
@@ -541,9 +567,13 @@ def set_node_network_state(self, node_address: Address, network_state: str):
         self.handle_state_change(state_change)
 
     def start_health_check_for(self, node_address: Address):
-        # This function is a noop during initialization. It can be called
-        # through the alarm task while polling for new channel events.  The
-        # healthcheck will be started by self.start_neighbours_healthcheck()
+        """Start health checking `node_address`.
+
+        This function is a noop during initialization, because health checking
```

Comment:
This is not true anymore because of the flag (old `dispatch_events_lock`), but I left it out anyways because at the moment it does no harm. An alternative is to assert on the state instead of making it a noop.